src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -464,7 +464,7 @@ def validate_max_surge(namespace):
             raise CLIError("--max-surge must be positive")
     except ValueError:
         raise CLIError("--max-surge should be an int or percentage")
-
+    
```

Comment:
remove extra whitespaces

Reply:
thanks suprised its not in linter though